File: hw/viteri/u2/HW18ZooInPython/Zoo/model/Shark.py
from datetime import date
from model.Fish import Fish
from model.Cage import Cage

class Shark(Fish):
    def __init__(self, id, description, born_on_date, gender, cage, number_of_bones, vertebrate):
        super().__init__(id, description, born_on_date, gender, cage, number_of_bones, vertebrate)

File: hw/segarra/u2/HW18ZooInPythonAndJavaScript/ec.edu.zooinjavascript.model/Shark.js
import Fish from "./Fish.js";

export default class Shark extends Fish{
    constructor(id, description, bornOnDate, gender, cage, numberOfBones, vertebrate){
        super(id, description, bornOnDate, gender, cage, numberOfBones, vertebrate)
    }

    feed(){
        console.log('Feeding a shark with fish')
    }

}